cirq/optimizers/two_qubit_to_fsim_test.py

```diff
@@ -76,55 +76,55 @@ def test_decompose_xx_yy_into_two_fsims_ignoring_single_qubit_ops_fail():
 
 @pytest.mark.parametrize('obj,fsim_gate',
```

Comment:
I would still have some tests for the *via_b method to protect the functionality of the (now deprecated) method as it still has logic in there, which is now untested (though accidentally gets covered by the deprecation check).

Reply:
Restored testing of the old function in most cases. This way we test both old and new versions. The only case where I changed the test to test the new function is the test where an assert needs to be modified due to different length of the resulting circuit. This way removing deprecated function will be trivial.